src/components/modebar/manage.js

```diff
@@ -138,7 +138,7 @@ function getButtonGroups(gd, buttonsToRemove, buttonsToAdd) {
         addGroup(['hoverClosestGl2d']);
     }
     else if(hasCartesian) {
-        addGroup(['hoverClosestCartesian', 'hoverCompareCartesian']);
+        addGroup(['toggleSpikelines', 'hoverClosestCartesian', 'hoverCompareCartesian']);
```

Comment:
![peek 2017-04-07 17-09](https://cloud.githubusercontent.com/assets/6675409/24820252/1d377f9e-1bb6-11e7-84da-fdfab7cff752.gif)

Personally, I think I'd prefer placing this new icon first from the right. Thoughts?